rs/ic_os/config_types/src/lib.rs

```diff
@@ -85,6 +85,12 @@ pub struct ICOSSettings {
     /// The URL (HTTP) of the NNS node(s).
     pub nns_urls: Vec<Url>,
     pub use_node_operator_private_key: bool,
+    /// Whether SEV-SNP is enabled. This is configured when the machine is deployed.
```

Comment:
Optional nit:

```suggestion
    /// Whether SEV-SNP should be enabled. This is configured when the machine is deployed.
```

As the config is not source of truth. Especially if we add a way to enable SEV-SNP remotely via proposal.

Reply:
> As the config is not source of truth.

Isn't it? Note that in the first iteration, we'll use config.ini to enable SEV-SNP. Later, once we configure it via the NNS, this flag will be deprecated and ineffective.

Reply:
We could use `should` in the comment. Nevertheless, if this flag is enabled and the node cannot come up with SEV, it should not come up at all and we need to investigate. It should never come up without SEV just as a fallback.